kiosk/src/Utils/domUtils.ts

```diff
@@ -43,3 +43,9 @@ export const isInteractable = (
         isElementValid(element)
     );
 };
+
+export const focusElement = (el: HTMLElement | null | undefined): void => {
+    setTimeout(() => {
+        el?.focus({ focusVisible: true, preventScroll: true } as any);
+    }, 0);
```

Comment:
Why is the setTimeout needed here?

Reply:
It probably isn't needed. I was struggling with getting focus to work reliably on Xbox, but turns out the problem was elsewhere. This was a workaround I found for a similar problem seen in earlier browsers like IE. I will make a note to remove it.